File: beginner/java/1181.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        double[][] m = new double[12][12];
        double soma = 0.0;
        int n;
        char operacao;

        Scanner scanner = new Scanner(System.in);

        n = scanner.nextInt();
        operacao = scanner.next().charAt(0);

        for (int i = 0; i < 12; i++) {
            for (int j = 0; j < 12; j++) {
                m[i][j] = scanner.nextDouble();
            }
        }

        if (operacao == 'S') {
            for (int j = 0; j < 12; j++) {
                soma += m[n][j];
            }
            System.out.printf("%.1f\n", soma);
        } else if (operacao == 'M') {
            for (int j = 0; j < 12; j++) {
                soma += m[n][j];
            }
            System.out.printf("%.1f\n", soma / 12);
        }

        /*for (int i = 0; i < 12; i++) {
            for (int j = 0; j < 12; j++) {
                System.out.printf("%.1f\t", m[i][j]);
            }
            System.out.println();
        }*/

        scanner.close();
    }
}

File: beginner/c/1181.c
#include <stdio.h>

int main() {
    double m[12][12], soma = 0.0;
    int i, j, n;
    char operacao;

    scanf("%d", &n);
    scanf(" %c", &operacao);

    for (i = 0; i < 12; i++) {
        for (j = 0; j < 12; j++) {
            scanf("%lf", &m[i][j]);
        }
    }

    if(operacao == 'S'){
        for(j = 0; j < 12; j++){
            soma += m[n][j];
        }
        printf("%.1lf\n", soma);
    } else if(operacao == 'M'){
        for(j = 0; j < 12; j++){
            soma += m[n][j];
        }
        printf("%.1lf\n", soma/12);
    }

    /*for (i = 0; i < 12; i++) {
        for (j = 0; j < 12; j++) {
            printf("%lf\t", m[i][j]);
        }
        printf("\n");
    }*/

    return 0;
}
